style/style.css

```diff
@@ -320,3 +333,210 @@ textarea {
 .btn-contact {
   width: 48%;
 }
+
+/*
+      ==============================================
+       *** MEDIA QUERY - Desktop Version- 1024px ***
+      ==============================================
+*/
+
+@media only screen and (min-width: 1024px) {
+  /* ------- Main ------- */
+  main {
+    background-image: url("../images/header-shapes-desktop.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    padding: 288px 24px 255px 24px;
+    border-bottom-left-radius: 150px;
+  }
+
+  header nav {
+    display: flex;
+  }
+
+  header .my-logo {
+    margin-left: 154px;
+  }
+
+  header nav .fa-bars {
+    display: none;
+  }
+
+  header nav ul {
+    display: flex;
+    gap: 52px;
+    padding-right: 154px;
+  }
+
+  header nav ul li a {
+    color: #344563;
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .section-top {
+    margin: 0;
+    padding: 359px 260px 255px 260px;
```

Comment:
- [ ] This padding-top in addition to the previous one, are too big

Reply:
Solved